pkgs/applications/misc/obinskit/default.nix

```diff
@@ -0,0 +1,81 @@
+{ lib
+, stdenv
+, fetchzip
+, patchelf
+, xlibs
+, glibc
+, gtk3-x11
+, alsaLib
+, nss
+, libpulseaudio
+, libxkbcommon
+, at_spi2_atk
+, makeWrapper
+, atomEnv
+}:
+
+let
+  libPath = lib.makeLibraryPath [
+    glibc
+    gtk3-x11
+    xlibs.libXtst
+    xlibs.libXt
+    xlibs.libXScrnSaver
+    alsaLib
+    nss
+    libpulseaudio
+    libxkbcommon
+    at_spi2_atk
+  ];
+
+in stdenv.mkDerivation rec {
+  pname = "obinskit";
+  version = "1.1.1";
+
+  src = fetchzip {
```

Comment:
:)
```suggestion
  src = fetchurl {
```

Reply:
I was told `fetchzip` works with archives beyond Zip and indeed it seems to work. Is it still better to use `fetchurl` and then adding an `unpackPhase`?

Reply:
probably doesn't matter in the end, but you're fetching a tarball, so I would say that a fetchurl is more expected.